package.json

```diff
@@ -43,6 +43,7 @@
     },
     "devDependencies": {
         "@comet/dev-process-manager": "^2.3.2",
+        "@comet/eslint-plugin": "workspace:^4.1.0",
```

Comment:
Why do I need this in the the root package.json?

Reply:
Because eslint packages get hoisted automatically (see https://pnpm.io/npmrc#public-hoist-pattern) in ordner to make eslint loading mechanisms work correctly.

And somehow this automatic hoisting doesn't work for workspace: dependencies, so I decided to manually add it to the root package.json.